dojo/tools/dependency_check/parser.py

```diff
@@ -196,6 +200,10 @@ def get_finding_from_vulnerability(self, dependency, related_dependency, vulnera
         # startswith CWE-XXX
         stripped_name = re.sub(r'^CWE-\d+', '', stripped_name).strip()
 
+        if component_name is None:
+            logger.error("component_name was None!")
+            return None
+
```

Comment:
ERROR log level is reserved for serious flaws that need to be looked into by the sysadmin. so this should be warning at most. and if you want it to be more useful I would add some context info about the finding that is being parsed or something relevant.

Reply:
It's now logging as warning. I also changed the logic to use `dependency_filename` when `component_name` cannot be found. It works really good. It doesn't drop the finding. Thanks for flagging this.